server/vulnerabilities/nvd/sync.go

```diff
@@ -277,11 +278,13 @@ func LoadCVEMeta(logger log.Logger, vulnPath string, ds fleet.Datastore) error {
 		meta = append(meta, score)
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
-	defer cancel()
-
-	if err := ds.InsertCVEMeta(ctx, meta); err != nil {
-		return fmt.Errorf("insert cve meta: %w", err)
+	// CVE Meta is only used for premium access patterns
+	if license.IsPremium(ctx) {
```

Comment:
Should we do a `license.IsPremium()` check at the beginning of the method and do nothing if it's not premium?

Reply:
Yeah good call, all this function does is prepare that dataset to be upserted, thanks!